TheFirstHomeTask.playground/Contents.swift

```diff
@@ -1,3 +1,35 @@
-import UIKit
+// Sum of separately whole and separately fractional parts of numbers
+var a = 2.5
+var b = 9.7
+var c = 6.9
+var d = 8.2
```

Comment:
Maybe it's better to rename these properties to `firstNumber`, `secondNumber` and so on for better readability?

Also, I think we can use `let` instead of `var` here because these values aren't changed along the way